tutorial/10_key_features/005_visualization.py

```diff
@@ -0,0 +1,163 @@
+"""
+.. _visualization:
+
+Quick Visualization
+===================
+
+Optuna provides various visualization features in :ref:`visualization_module` to interpret hyperparameter relatioships visually.
+
+This tutorial walks you through this module by visualizing the history of multi-layer preceptron for FashionMNIST implemented in PyTorch.
+
+This module is implemented with `plotly <https://plotly.com>`_ but `matplotlib <https://matplotlib.com>`_ backend
+is under construction as of September, 2020.
```

Comment:
IMHO, this information can be a note not to stray from the main point.

Reply:
So, do you mean this information can be omitted?